lookup_tables/greynoise/advanced/noise_advanced.yml

```diff
@@ -102,6 +102,19 @@ LogTypeMap:
         - "LocalAddressIP6"
         - "RemoteAddressIP4"
         - "RemoteAddressIP6"
+    - LogType: Crowdstrike.FDREvent
+      Selectors:
+        - 'p_any_ip_addresses'
+        - '$.event.CurrentLocalIP'
```

Comment:
I think there's a potential overhead in trying so many selectors. I think we cover several of these fields with our IP extraction logic, but some are definitely missing (e.g `UserIp`, `LocalIP`, `GatewayIP`, `CurrentLocalIP` - actually that should be all of them 😄 ). 


We could add them on the parser instead and just use `p_any_ip_addresses` here, what do you think?

Reply:
Yup, wanted to talk about this too. Asked investgations too. I second this. We should do it in the parser

Reply:
Nice, thanks, that'll improve Indicator Search results of course 😄